pkg/libinit/proc_plan9.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2014-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package libinit
+
+import (
+	"log"
+	"os"
+	"os/exec"
+	"syscall"
+)
+
+// WaitOrphans waits for all remaining processes on the system to exit.
+func WaitOrphans() uint {
+	var numReaped uint
+	for {
+		var (
+			w syscall.Waitmsg
+		)
+		err := syscall.Await(&w)
+		if err != nil {
+			break
+		}
+		log.Printf("Exited with %v", w)
+		numReaped++
+	}
+	return numReaped
+}
+
+// WithCloneFlags adds clone(2) flags to the *exec.Cmd.
+func WithCloneFlags(flags uintptr) CommandModifier {
```

Comment:
I'd get rid of this altogether and make a new WithPIDNamespace, if pid namespaces exist in p9 (I don't remember)

Reply:
they don't ... but you do have lots of flags on rfork. So it pays to just have it.

We should probably get some name other than Clone, since the same thing on freebsd is called rfork, as on plan 9

WithForkFlags might be good enough?

Reply:
If they can't share the flag values themselves, it's probably also confusing to share the name, so I'd keep them deliberately separate.

Reply:
done. Name changed.